src/operator/src/statement/ddl.rs

```diff
@@ -1224,6 +1228,13 @@ impl StatementExecutor {
     }
 }
 
+/// Fills schema-level options to [CreateTableExpr].
+fn fill_schema_opts(expr: &mut CreateTableExpr, schema: &SchemaNameValue) {
+    if let Some(ttl) = schema.human_ttl() {
+        expr.table_options.insert(TTL_KEY.to_string(), ttl);
```

Comment:
Will a ttl key in `expr.table_options` be overridden if it already exists?

Reply:
> Will a ttl key in `expr.table_options` be overridden if it already exists?

This has been addressed in d033ae8bbb